torch/csrc/api/include/torch/special.h

```diff
@@ -161,8 +161,40 @@ inline Tensor i0e(const Tensor& self) {
   return torch::special_i0e(self);
 }
 
-inline Tensor i0e_out(const Tensor& self) {
-  return torch::special_i0e(self);
+inline Tensor i0e_out(Tensor& result, const Tensor& self) {
```

Comment:
Missed out on this in the previous PR.

Have added a very basic test in `special.cpp`

Reply:
Would you move this fix to a separate PR? I don't know if we need a test for this

Reply:
Have kept the fix. But removed the test. Is it okay?

Reply:
Sounds good ;)